core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeService.java

```diff
@@ -279,6 +288,16 @@ public String generateShortcode() {
         return shortcode;
     }
 
+    public Optional<Enrollee> findOneByParticipantUserIdAndStudyEnvironmentId(UUID participantUserId, UUID studyEnvId) {
+        return dao.findByParticipantUserIdAndStudyEnvId(participantUserId, studyEnvId);
+    }
+
+    public Optional<Enrollee> findOneByParticipantUserIdAndStudyEnvironmentId(UUID participantUserId, String studyShortcode, EnvironmentName envName) {
```

Comment:
a little shorter, and less misleading since we're not finding by studyEnvironmentId
```suggestion
    public Optional<Enrollee> findByParticipantUserIdAndStudyEnv(UUID participantUserId, String studyShortcode, EnvironmentName envName) {
```